src/lightning_app/cli/cmd_clusters.py

```diff
@@ -130,22 +149,30 @@ def create(
                 click.echo("cluster unchanged")
 
         resp = self.api_client.cluster_service_create_cluster(body=new_body)
-        if wait:
-            _wait_for_cluster_state(self.api_client, resp.id, V1ClusterState.RUNNING)
-
         click.echo(
             dedent(
                 f"""\
-            {resp.id} is now being created... This can take up to an hour.
+            BYOC cluster creation triggered successfully!
+            This can take up to an hour to complete.
 
             To view the status of your clusters use:
-                `lightning list clusters`
+                lightning list clusters
 
             To view cluster logs use:
-                `lightning show cluster logs {resp.id}`
-                    """
+                lightning show cluster logs {cluster_name}
+
+            To delete the cluster run:
+                lightning delete cluster {cluster_name}
+            """
             )
         )
+        if not do_async:
+            try:
+                _wait_for_cluster_state(self.api_client, resp.id, V1ClusterState.RUNNING)
+                return
+            except KeyboardInterrupt:
+                pass
+        click.echo("\nCluster will be created in the background!")
```

Comment:
let's put this message under the except clause and remove the return above

Reply:
I'll try to make this easier to follow / maintain.

I want to say "Cluster will be created in the background" if 1. we're running async, or 2. we were waiting but then hit `<ctrl-c>`.

I don't want to print the message if we wait long enough for the cluster to complete.